README.md

```
@@ -1,3 +1,52 @@
+### 입출력
+
+- 플레이어 두 명의 이름을 입력받는다.
+    - 플레이어 두 명의 이름을 입력하시오. (플레이어1, 플레이어2)
+- 입력 순서대로 각 플레이어의 팀을 지정하여 출력한다.
+    - 플레이어1은 청팀, 플레이어2는 홍팀입니다.
+- 게임 시작 시 장기판의 초기 상태를 보여준다.
+- 게임 시작 메시지를 출력한다
+    - 게임을 시작합니다.
+- 움직일 좌표를 입력받는다.
+    - 청팀은 움직일 말의 좌표와 이동할 좌표를 입력하세요.
+- 한 턴마다 새로운 장기판을 보여준다.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)